### PR TITLE
Change dataset and update README

### PR DESCRIPTION
**Pull request by xiaowang1105**

### CODE REVIEW

**Comment by shane-huang** [comment]
1.
> ..., i.e. number of NYC taxi passengers, which contains 10320 taxi passengers usage values in time sequence
to 
> ...,  i.e. NYC taxi passengers dataset, which contains 10320 records, each record indicating the total number of taxi passengers in NYC at a corresonponding time spot.
2. The figure in block [7] looks too tight and the curve is not clear enough. Make the curve line thinner or change the width of the figure. Same for figure in block [24]
3. The y axies of figure in block [9] is "Number", not clear explanation, maybe change it to "record counts". And I think this figure is actually a histogram with 2 cateogories plotted in the same figure. You can just say it's a "histogram" in figure title. 
4. In block [12], change
>ie if unroll_length = 5
to 
>for example, if unroll_length=5

File: notebook/python/anomaly-detection/README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ This is an simple example of unsupervised anomaly detection using Zoo Keras API.
 ## Run with Jupyter
 * Download ZOO and build it. (TODO: to add a link to download and build page after our website is set up.)
```
Comment (shane-huang) [inline notebook/python/anomaly-detection/README.md]: remove TODO.

File: data/nyc_taxi/get_nyc_taxi.sh [hunk 2]
```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
```
Comment (shane-huang) [inline data/nyc_taxi/get_nyc_taxi.sh]: change the data folder from `data/nyc_taxi` to `data/NAB/nyc_taxi`

File: notebook/python/anomaly-detection/README.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
 # Anomaly Detection
-This is an simple example of unsupervised anomaly detection using Zoo Keras API. We use RNN to predict following data values based on previous sequence (in order) and measure the distance between predicted values and actual values. If the distance is above some threshold, we report those values as anomaly. This example is modified from https://github.com/Vicam/Unsupervised_Anomaly_Detection
+This is a simple example of unsupervised anomaly detection using Zoo Keras API. We use RNN to predict following data values based on previous sequence (in order) and measure the distance between predicted values and actual values. If the distance is above some threshold, we report those values as anomaly.
```
Comment (jason-dai) [inline notebook/python/anomaly-detection/README.md]: Just "using Keras API"

**Comment by shane-huang** [comment]
LGTM. It can be merged after passing the PR validation.

**Comment by xiaowang1105** [comment]
jenkins passed http://172.168.2.101:8080/view/ZOO-NB-PIPELINE/job/ZOO-NB-TutorialTests/56/

**Pull request merged**